docs/database.rst

```diff
@@ -0,0 +1,40 @@
+rotki Database
+##############
+
+rotki uses two different sqlite databases, one with information about assets (global.db) and one with user information (rotkehlchen.db).
+The latter is encrypted using an extension called `SQLCipher <https://github.com/sqlcipher/sqlcipher>`__
+that provides transparent 256-bit AES full database encryption.
+
+Database Location
+=================
+
+The default database locations are:
+
+- **Linux**: ``~/.share/local/rotki/data``
+- **OSX**: ``??``
+- **Windows**: ``??``
+
+Exploring the database
+**********************
+
+To open the user database you can use `sqlitebrowser <https://sqlitebrowser.org/>`__.
+It supports sqlcipher and will ask for the password used to decrypt the database.
+
+If you prefer the command line instead, you can use the ``sqlcipher`` cli tool. Note: using the ``sqlite3`` cli will not work!
+
```

Comment:
```suggestion

If you prefer the command line instead, you can use the ``sqlcipher`` cli tool. Note: using just ``sqlite3`` cli will not work since the database is encrypted.

```